exercises/ex2.cpp

```diff
@@ -13,39 +13,8 @@ int main()
 
   cout << "Enter week number(1-7): " << endl;
   cin >> week;
-
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
-    cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
-    cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
-  }
+  string arr[7] = {"Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"};
+  cout<<arr[week]<<endl;
```

Comment:
Dovresti controllare che il valore in input sia fra [0, 6], che fra l'altro non corrisponde con quello richiesto dal programma fra [1, 7],